packages/create-block-app/create-block-app.js

```diff
@@ -75,52 +73,53 @@ const usage = commandLineUsage(helpSections);
   const resolvedBlockPath = path.resolve(untildify(folderPath));
 
   try {
-    fs.statSync(resolvedBlockPath);
+    await fs.stat(resolvedBlockPath);
     console.error(
       `${resolvedBlockPath} already exists, please specify another path!`,
     );
     process.exit();
   } catch {
-    // noop (we expect statSync to fail)
+    // noop (we expect stat to fail)
   }
 
   const tempExtractionDir = path.join(resolvedBlockPath, "tmp");
 
-  console.log("Downloading template...");
+  const templatePackageName = `block-template-${template}`;
+  console.log(`Downloading package ${templatePackageName}...`);
 
   await pacote.extract(templatePackageName, tempExtractionDir, {
     registry: process.env.NPM_CONFIG_REGISTRY,
   });
 
-  const templatePath = path.join(tempExtractionDir, "templates", template);
+  console.log("Updating files...");
+
+  await fs.copy(
+    path.resolve(tempExtractionDir),
+    path.resolve(resolvedBlockPath),
+  );
+  await fs.rm(tempExtractionDir, { recursive: true });
+
   try {
-    fs.statSync(templatePath);
-  } catch {
-    console.error(
-      `Template '${template}' is missing – please raise an issue at https://github.com/blockprotocol/blockprotocol/issues`,
+    await fs.rename(
+      path.resolve(resolvedBlockPath, ".gitignore.dist"),
+      path.resolve(resolvedBlockPath, ".gitignore"),
```

Comment:
Restores logic from https://github.com/blockprotocol/blockprotocol/pull/102 (removed in https://github.com/blockprotocol/blockprotocol/pull/348)

We need to include `.gitignore.dist` because `.gitignore` does not get published.